src/runtime-rs/crates/runtimes/src/manager.rs

```diff
@@ -370,7 +370,21 @@ impl RuntimeHandlerManager {
             state: state.clone(),
         };
 
-        inner.try_init(sandbox_config, Some(spec), options).await
+        // Load config to check hypervisor settings
+        let config = load_config(&sandbox_config.annotations, options).context("load config")?;
+        let mut amended_spec = spec.clone();
+        if let Some(hypervisor_config) = config.hypervisor.get(&config.runtime.hypervisor_name) {
+            if hypervisor_config.disable_guest_selinux {
+                if let Some(ref mut process) = amended_spec.process_mut() {
+                    process.set_selinux_label(None);
+                }
+                if let Some(ref mut linux) = amended_spec.linux_mut() {
+                    linux.set_mount_label(None);
+                }
```

Comment:
@mikn There's already code that does that at https://github.com/kata-containers/kata-containers/blob/main/src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs#L636 , pretty much like with the go runtime. Isn't it called in your case ?